scripts/test-integration.sh

```diff
@@ -49,7 +49,6 @@ else
 
 	# Configuration for more verbose output
 	export VSCODE_CLI=1
-	export ELECTRON_ENABLE_STACK_DUMPING=1
```

Comment:
why?

Reply:
This flag belonged to a merge that was done 3 years ago. I have removed it because vscode is not using it anymore.

Reply:
I'm guessing this was removed because it apparently does nothing if the crash reporter is enabled (which it is during tests) 

https://www.electronjs.org/docs/latest/api/environment-variables#electron_enable_stack_dumping